examples/AzureML-REST-API/submit-custom-image.md

```diff
@@ -78,12 +78,49 @@ resp = requests.post(start_run, files=run_files, headers=header)
 print("response text: ", json.loads(resp.text))
 ```
 
+## Tag the run
+
+Tagging runs lets you later retrieve runs by more meaningful identifiers than the run ID.
+
+```python
+run_id = resp.json()["runId"]
+
+run_uri = hosturl + historybase + resourcebase + "experiments/{}/runs/{}".format(experiment_name,run_id)
+payload = {
+   "tags": {
+      "customimage": "specialrun"
+   }
+}
+
+# If you forget the content type header, you'll get an HTTP 415
+header.update({
+    "content-type": "application/json"
+})
+resp = requests.patch(run_uri, json=payload, headers=header)
+print(resp.json()["tags"])
+```
+
+## Retrieve a run by tag
+
+Later, you can fetch the run by querying the experiment
+
+```python
+query_runs = hosturl + historybase + resourcebase + "experiments/{}/runs:query".format(experiment_name)
+payload = {
+   "filter": "Tags/customimage eq specialrun",
```

Comment:
How does this filtration syntax work exactly? is it documented?

Reply:
It's Odata with a slight tweak for nested properties. You're right that it's probably not documented sufficiently somewhere. We should do that eventually